qCC/ccCommandLineCommands.h

```diff
@@ -27,6 +27,13 @@ struct CommandChangeMeshOutputFormat : public CommandChangeOutputFormat
 	bool process(ccCommandLineInterface& cmd) override;
 };
 
+struct CommandChangeHierarchyOutputFormat : public CommandChangeOutputFormat
```

Comment:
Style: I would make this a `class`. I think we typically use `struct` only for POD.

 (Happy to discuss - not a strong opinion)

Reply:
All commands are structs